twoliter/embedded/build.Dockerfile

```
@@ -157,9 +167,12 @@ RUN --mount=source=.cargo,target=/home/builder/.cargo \
     --mount=type=cache,target=/home/builder/.cache,from=cache,source=/cache \
     --mount=type=cache,target=/home/builder/rpmbuild/BUILD/sources/models/src/variant,from=variantcache,source=/variantcache \
     --mount=source=sources,target=/home/builder/rpmbuild/BUILD/sources \
+    # '-' is an illegal char in RPM Release field; -dirty commits become _dirty
+    BUILD_ID=$(echo ${BUILD_ID} | tr '-' '_') ; \
     rpmbuild -ba --clean \
       --undefine _auto_set_build_flags \
       --define "_target_cpu ${ARCH}" \
+      --define "dist ${BUILD_ID_TIMESTAMP}.${BUILD_ID}.br1" \
```

I'm usually the one that complains about "sanitizing" `-dirty` but here I would do that instead:
```suggestion
      --define "dist ${BUILD_ID_TIMESTAMP}.${BUILD_ID//-dirty/}.br1" \
```

We don't rebuild RPMs when the build ID changes, so for local development this would make most packages always have the "-dirty" release tag. That's going to irk me because it will show up in the RPM file name even if that package doesn't have any uncommitted changes in my checkout.

---

> That's going to irk me because it will show up in the RPM file name even if that package doesn't have any uncommitted changes in my checkout.

Yea that's a fair point. I'll take your suggestion :)